cxmon/src/mon.h

```diff
@@ -97,4 +102,7 @@ extern void mon_write_half(uintptr adr, uint32 w);
 extern uint32 mon_read_word(uintptr adr);
 extern void mon_write_word(uintptr adr, uint32 l);
 
+// Check if break point is set
+#define isBreakPoint(address) (active_break_points.find(address) != active_break_points.end())
```

Comment:
Use IS_BREAK_POINT() convention since this is a macro.

Reply:
Done!